app/uk/gov/hmrc/apiplatformdeskpro/scheduled/CreatePersonService.scala

```diff
@@ -42,8 +42,28 @@ class CreatePersonService @Inject() (
   )(implicit val ec: ExecutionContext
   ) extends ApplicationLogger with ClockNow {
 
-  // Not required currently. Retain for future reference
-  //
+  implicit val hc: HeaderCarrier = HeaderCarrier()
+  val daysToLookBackInNormalRun  = 31
+
+  def pushNewUsersToDeskpro()(implicit ec: ExecutionContext): Future[Unit] = {
+    for {
+      users          <- developerConnector.searchDevelopers(if (config.initialImport) None else Some(daysToLookBackInNormalRun))
```

Comment:
Could we extract the Option days to look back outside of the for comp? 
And could we do the logging in the connector?

Reply:
Ok, that's changed